### PR TITLE
test: update ACs for system tests

### PR DESCRIPTION
**Pull request by C0deMunk33**

### CODE REVIEW

File: protocol/0030-ETHM-multisig_control_spec.md [hunk 1]
```diff
@@ -127,3 +127,48 @@ contract MultisigControl {
 ### MultisigControl Consuming Smart Contract
 * Consuming smart contract calls `verify_signatures` with valid signature bundle and message hash is returned true if the valid signature count is over threshold % of total signers  (<a name="0030-ETHM-023" href="#0030-ETHM-023">0030-ETHM-023</a>)
  
+## Multisig Control to Vega Integration Tests (Vega System Tests)
+1. `mapping(address => bool) public signers;`
```
Comment (davidsiska-vega) [inline protocol/0030-ETHM-multisig_control_spec.md:133]: Is this mapping to be queried on Ethereum? Or some info the data-node holds?

File: protocol/0030-ETHM-multisig_control_spec.md [hunk 2]
```diff
@@ -127,3 +127,48 @@ contract MultisigControl {
...
+   * must show "True" for each signer that is currently valid (<a name="0030-ETHM-024" href="#0030-ETHM-024">0030-ETHM-024</a>)
```
Comment (davidsiska-vega) [inline protocol/0030-ETHM-multisig_control_spec.md:134]: What is the meaning of signer being valid from Vega chain side? That they're one of the "consensus forming" validators?

File: protocol/0031-ETHB-ethereum_bridge_spec.md [hunk 3]
```diff
@@ -148,3 +148,98 @@ This example connects the network to Ropsten:
 * ERC20 smart contract specific requirements:
   * A valid multisig bundle can be passed to the `remove_asset` function to successfully remove a previously  allow listed token (<a name="0031-COSMICELEVATOR-019" href="#0031-COSMICELEVATOR-019">0031-COSMICELEVATOR-019</a>)
   * An invalid multisig bundle is rejected by the `remove_asset` function (<a name="0031-COSMICELEVATOR-020" href="#0031-COSMICELEVATOR-020">0031-COSMICELEVATOR-020</a>)
+
+
+## ERC20 Bridge Logic to Vega Integration Tests (Vega System Tests)
+1.  `address payable public erc20_asset_pool_address;`
```
Comment (davidsiska-vega) [inline protocol/0031-ETHB-ethereum_bridge_spec.md:156]: Again, I am guessing this is to be verified on the ethereum side (would be good to clarify).

File: protocol/0031-ETHB-ethereum_bridge_spec.md [hunk 4]
```diff
@@ -148,3 +148,98 @@ This example connects the network to Ropsten:
...
+   * must match the deployed asset pool address (<a name="0031-ETHB-018" href="#0031-ETHB-018">0031-ETHB-018</a>)
+2.  `function list_asset(address asset_source,bytes32 vega_asset_id,uint256 lifetime_limit,uint256 withdraw_threshold,uint256 nonce,bytes memory signatures)`
+   * must list asset (<a name="0031-ETHB-019" href="#0031-ETHB-019">0031-ETHB-019</a>)
+   * must not list non-asset (<a name="0031-ETHB-020" href="#0031-ETHB-020">0031-ETHB-020</a>)
+   * must not list if sigs bad (<a name="0031-ETHB-021" href="#0031-ETHB-021">0031-ETHB-021</a>)
+   * must not list if already listed (<a name="0031-ETHB-022" href="#0031-ETHB-022">0031-ETHB-022</a>)
+3.  `function remove_asset(address asset_source,uint256 nonce,bytes memory signatures)`
```
Comment (davidsiska-vega) [inline protocol/0031-ETHB-ethereum_bridge_spec.md:163]: What happens with assets "locked" on the bridge if this is called?

**Review by davidsiska-vega — COMMENTED** [review]
Left some comments.

File: protocol/0030-ETHM-multisig_control_spec.md (hunk 1 above)
Comment (C0deMunk33) [reply protocol/0030-ETHM-multisig_control_spec.md:133]: it's on Ethereum, the `public` makes it act like a `view` when queried `singers(signer_address)`
The test should compare against the list of validators in core

File: protocol/0030-ETHM-multisig_control_spec.md (hunk 2 above)
Comment (C0deMunk33) [reply protocol/0030-ETHM-multisig_control_spec.md:134]: yes, that the validators that core knows about return `True` when the contract signers are queried

File: protocol/0031-ETHB-ethereum_bridge_spec.md (hunk 3 above)
Comment (C0deMunk33) [reply protocol/0031-ETHB-ethereum_bridge_spec.md:156]: it is verified on chain, but against what the system tests have for those contracts

File: protocol/0031-ETHB-ethereum_bridge_spec.md (hunk 4 above)
Comment (C0deMunk33) [reply protocol/0031-ETHB-ethereum_bridge_spec.md:163]: removing asset only stops deposits

**Review by davidsiska-vega — DISMISSED** [review]
LGTM though a round by e.g. @jgsbennett / @vega-paul would be good.

**Review by vega-paul — APPROVED** [review]

**Pull request merged**